toc/rfc/rfc-draft-pcap-bosh.md

```diff
@@ -2,60 +2,214 @@
 
-The main goal of the pcap feature is to allow users (BOSH operator in this case) to easily capture network traffic across multiple VMs by specifying various parameters in PCAP CLI:
-- BOSH deployment
-- BOSH instance group(s)
-- BPF [Berkeley Packet Filter](https://www.tcpdump.org/manpages/pcap-filter.7.html) filter to apply
+## Problem
 
-PCAP CLI will form a capture request and send it to PCAP API. The capture will then take place in real-time similar to running [tcpdump](https://www.tcpdump.org/manpages/tcpdump.1.html) in a distributed fashion across multiple machines. The PCAP API will contact PCAP AGENT on all nodes involved and trigger a network capture. The PCAP AGENTs will concurrently stream all captured traffic back to PCAP AGENT where all streams are merged into one contiguous stream that is sent back to the PCAP CLI where it is then stored as a PCAP FILE on disk.
+Distributed, and particularly networked systems are hard to debug. Software developers and operators are not necessarily the same people, in some cases not even the same companies.
 
-All communication between PCAP CLI, PCAP API and PCAP AGENT(s) is carried out via gRPC over HTTP/2.
+Network traffic can be reasonably easily captured via tools such as `tcpdump` on local machines. In order to get a unified view of the network traffic of relevant nodes, those captures need to be retrieved, merged and analysed.
 
-Trust is established as follows:
-1. PCAP CLI uses the BOSH CLI `access_token` and sends them to PCAP API
-2. PCAP API verifies `access_token` against BOSH UAA, verifies the token scope and establishes mTLS with PCAP AGENT
-3. PCAP AGENT only accepts mTLS connections from PCAP AGENT via a trusted CA
+BOSH does not have a user or operator friendly way for resolving targets, or capturing, filtering and merging network traffic.
 
-## Problem
+## Proposal
 
-The current architecture for "PCAP for BOSH" is depicted as follows:
+The feature proposed in this RFC has two possible solutions, each of which have their benefits and drawbacks. The solution with the most community support will ultimately be implemented. Motivations, constraints and considerations are highlighted in those respective sections.
 
-![PCAP BOSH Architecture](rfc-draft-pcap-bosh/tcpdump-for-bosh.svg)
+The following solutions are discussed in subsequent sections (linked below):
+1. [Integrate pcap-release with BOSH](#integrate-pcap-release-with-bosh-cli-director-agent)  
+   This solution focuses on integrating components that are reusable for other scenarios and ensure long-term availability, use and improvements.
+2. [Add "pcap-lite", a thin front-end and merge functionality of tcpdump streams](#add-pcap-lite-a-thin-front-end-and-merge-functionality-of-tcpdump-streams)  
+   This solution focuses on a convenient minimal solution for the problem statement, but presents a separate code base.
 
+> **Warning**
```

Comment:
I don't get why a warning is needed. 
```suggestion
> **Remark**
```
?

Reply:
`Remark` doesn't exist, you can have [Note, Important or Warning](https://github.com/orgs/community/discussions/16925)

Reply:
Also thanks for the link. Apparently the new syntax is `[!NOTE]`

> [!NOTE]
> This is a note.